app/src/App.tsx

```diff
@@ -16,13 +14,13 @@ const App = () => {
    *
    * When a React component is unmounted, it will invoke the return value of any
    * `useEffect` functions. In this case, if the `App` component is unmounted
-   * then the `abort` function is called so the `bootstrapApp` action can no
+   * then the `abort` functions are called so the `bootstrap` actions can no
```

Comment:
Why change the wording here? Looks like you still only do `.abort` 1 time in this file?

Reply:
There will be future bootstrap actions, such as `bootstrapCallInvites` and `bootstrapCalls`.